sova/cli/commands/briefing.py

```
@@ -0,0 +1,126 @@
+
+    try:
+        if unit == "h":
+            delta = timedelta(hours=value)
+        elif unit == "m":
+            delta = timedelta(minutes=value)
+        elif unit == "d":
+            delta = timedelta(days=value)
+        else:
+            return None
+
+        return datetime.now() - delta
```

Ruff (0.16.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)